### ISSUE TITLE
sed: can't read [...]: No such file or directory

### ISSUE THREAD

**Issue by PWNL506** [vinceliuice/WhiteSur-icon-theme#149]
Hi, 
Tried to install the Icon Theme, and I am pretty new to that kinda stuff. When I tried to install it via Terminal, multiple errors occurred (picture). The errors go on for 3 more pages..
Can anyone help?
![Screenshot from 2022-03-18 10-40-48](https://user-images.githubusercontent.com/94864177/158979023-e5cac2b0-1633-4a45-ae38-9496c69aa36e.png)

**Comment by vinceliuice** [comment]
Before you unzip this zip file you need to make sure p7zip-full installed, because without this linux can not full support zip files that'll make some files missing, or you can download this: https://github.com/vinceliuice/WhiteSur-icon-theme/archive/refs/tags/2022-03-18.tar.gz

**Comment by gvaishno** [comment]
Was just trying to install this pack. Facing same issue. In my case p7zip-full is already installed and latest.

**Comment by vinceliuice** [comment]
What's your system version @gvaishno

**Comment by gvaishno** [comment]
> What's your system version @gvaishno

Distributor ID: Elementary
Description: elementary OS 6.1 Jólnir
Release: 6.1
Codename: jolnir
Kernel version: 5.13.0-35-generic

I was able to resolve it by git cloning the repository instead of downloading the source code.

**Comment by vinceliuice** [comment]
Linux not fully support zip files so if you unzip it it'll loss some files

**Comment by gvaishno** [comment]
> Linux not fully support zip files so if you unzip it it'll loss some files, that's why cloneing it can work, you can install p7zip p7zip-full p7zip-rar to fix it

You are right. I think we should add the "How to extract zip" or similar section in the README.md along with the prerequisites which you mentioned.
